### ISSUE TITLE
IntegrationTests merge folders/suites into one 

### ISSUE THREAD

**Issue by Dreaky** [apideck-libraries/portman#361]
Hi, I tried to create more than only one integration test and always I get only one test suite, maybe I don't understand correctly but in the scheme is defined:  
```
"integrationTests": {
    "items": {
        "additionalProperties": false,
        "properties": {...},
        "required": [...],
        "type": "object",
    },
    "type": "array"
},
```
and what I want is something like this: 
```
"integrationTests": [
    {
        "name": "suites_no1",
        "operations": [....]
    },
    {
        "name": "suites_no2",
        "operations": [....]
    },
    {
        "name": "suites_no3",
        "operations": [....]
    },
```
I have this definition in my portmanConfigFile ( tried both json and yaml ) 
After generate postman collection all operations were merged into the first suite e.g. suites_no1

**Comment by Dreaky** [comment]
Hello, 
I was able to reproduce the same bug using your example API:  
https://github.com/apideck-libraries/portman/tree/main/examples/portman-globals

I append my integration tests to `portman-config.crm.json` 

```
{
    "version": 1.0,
    "tests": {
        "integrationTests": [
            {
                "name": "suites_no1",
                "operations": [
                    {
                        "openApiOperationId": "leadsAdd",
                        "variations": [
                            {
                                "name": "Add in SUITE 01",
                                "tests": {
                                    "contractTests": [
                                        {
                                            "statusSuccess": {
                                                "enabled": true
                                            }
                                        },
                                        {
                                            "responseTime": {
                                                "enabled": true,
                                                "maxMs": 1000
                                            }
                                        },
                                        {
                                            "schemaValidation": {
                                                "enabled": true
                                            }
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                ]
            },
            {
                "name": "suites_no2",
                "operations": [
                    {
                        "openApiOperationId": "leadsAll",
                        "variations": [
                            {
                                "name": "All in SUITE 02",
                                "tests": {
                                    "contractTests": [
                                        {
                                            "statusSuccess": {
                                                "enabled": true
                                            }
                                        },
                                        {
                                            "responseTime": {
                                                "enabled": true,
                                                "maxMs": 1000
                                            }
                                        },
                                        {
                                            "schemaValidation": {
                                                "enabled": true
                                            }
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                ]
            }
        ]
    },
    "globals": {
        "stripResponseExamples": true,
        "collectionPreRequestScripts": [
            "pm.collectionVariables.set('status', pm.iterationData.get('status') || 'open')"
        ],
        "collectionTestScripts": [
            "pm.collectionVariables.set('applicationId', pm.iterationData.get('applicationId') || '1111');"
        ],
        "keyValueReplacements": {
            "x-apideck-app-id": "{{applicationId}}"
        },
        "valueReplacements": {
            "<Bearer Token>": "{{bearerToken}}"
        },
        "rawReplacements": [
            {
                "searchFor": "Unify",
                "replaceWith": "Unify ApiDeck"
            }
        ]
    }
}
```

and result it was same as I describe upper 
![portman-folders-bug](https://user-images.githubusercontent.com/8088852/179188680-bf3cd3b4-96bc-4b28-b0df-53bd4880cebc.PNG)

**Comment by proper-function** [comment]
Hey guys, just letting you know I stumbled upon this as well. I can work around it though

**Comment by thim81** [comment]
hi @Dreaky & @proper-function,

We merged a PR from @nflcosta, that fixed this error.
The fix will be part of the next Portman release.

**Comment by thim81** [comment]
hi @Dreaky and @proper-function 

🎊 We just released Portman 1.21.0, which includes the fix that solves this folder issue.
